src/iexamine.cpp

```diff
@@ -4113,10 +4113,11 @@ void iexamine::autodoc( player &p, const tripoint &examp )
         amenu.ret > 1 ) {
         needs_anesthesia = false;
     } else {
-        std::vector<const item *> a_filter = p.crafting_inventory().items_with( []( const item & it ) {
+        const inventory &crafting_inv = g->u.crafting_inventory();
```

Comment:
Same here, I think this is trying to be NPC friendly
```suggestion
        const inventory &crafting_inv = u.crafting_inventory();
```